kvbc/test/multiIO_test.cpp

```diff
@@ -64,14 +64,13 @@ class multiIO_test : public ::testing::Test {
  protected:
   void SetUp() override {
     keyGen_.reset(new concord::kvbc::v1DirectKeyValue::RocksKeyGenerator);
-    comparator_ = new KeyComparator(new DBKeyComparator());
-    dbClient.reset(new Client(dbPath_, comparator_));
+    auto comparator_ = std::make_unique<KeyComparator>(new DBKeyComparator{});
```

Comment:
[Minor]: I think we can remove the `comparator_` variable and call `std::make_unique()` directly when constructing the `Client`.